Clients/GUI/Pages/UserInfo.razor

```diff
@@ -0,0 +1,133 @@
+﻿@page "/userinfo"
+@inject NavigationManager NavManager
+@inject IModalService Modal
+@attribute [Authorize]
+
+<div class="col-12">
+    <h3><b>User Info</b></h3>
+    <hr />
+    <EditForm Model="@user">
+        <DataAnnotationsValidator />
+        <div class="form-group">
+            <label for="Name">Name:</label>
+            <InputText class="input-group-text" disabled="@IsEditModeDisabled" @bind-Value="user.Name" />
+        </div>
+        <div class="form-group">
+            <label for="Surname">Surname:</label>
+            <InputText class="input-group-text" disabled="@IsEditModeDisabled" @bind-Value="user.Surname" />
+        </div>
+        <div class="form-group">
+            <label for="Birthday">Birthday:</label>
+            <InputDate class="input-group-text" disabled="@IsEditModeDisabled" @bind-Value="@user.DateOfBirth" />
+        </div>
+        <div class="form-group">
+            <label for="Email">Email:</label>
+            <InputText class="input-group-text" disabled="disabled" @bind-Value="@user.Email" />
+        </div>
+        <div class="form-group" style="@(IsPasswordBeingChanged ? "" : "height: 0; color: transparent")">
+            <label for="OldPassword">Enter old password: </label>
+            <InputText type="@(IsPasswordBeingChanged ? "password" : "hidden")" class="input-group-text" disabled="@(!IsPasswordBeingChanged)" @bind-Value="@user.OldPassword" />
+        </div>
+        <div class="form-group" style="@(IsPasswordBeingChanged ? "" : "height: 0; color: transparent")">
+            <label for="NewPassword">Enter new password: </label>
+            <InputText type="@(IsPasswordBeingChanged ? "password" : "hidden")" class="input-group-text" disabled="@(!IsPasswordBeingChanged)" @bind-Value="@user.NewPassword" />
+        </div>
+        <br />
+        <input type="@(IsEditModeDisabled ? "hidden" : "button")" @onclick="@ChangePasswordClickHandler" class="form-control col-3 button btn-primary" value="@(IsPasswordBeingChanged ? "Cancel changing password" : "Change password")" />
+        <br />
+        <ValidationSummary />
+        <h4 style="display:@(ErrorOccured ? "block" : "hidden"); color:red">@ErrorMessage</h4>
+        <input type="button" @onclick="@EditInfoClickHandler" class="form-control col-3 button btn-primary" value="@(IsEditModeDisabled ? "Edit" : "Save changes")" />
+    </EditForm>
+</div>
+
+@code {
+    public bool IsEditModeDisabled { get; set; }
+    public bool IsPasswordBeingChanged { get; set; }
+    public bool PopupVisible { get; set; }
```

Comment:
What is visible?

Reply:
You mean translation from English to Russian or what? :)

Reply:
Popup? I don't understand what is it mean

Reply:
> Popup? I don't understand what is it mean

It's a window, popping up inside a main window. I thought that such name is as fine as `ModalVisible `, but I can change it if you want

Reply:
ok, Thanks